en/docs/users/05_Configuration.md

```diff
@@ -1,31 +1,128 @@
-# Changing the view
+
+# Display
+
+## Language
+
+At the moment, FreshRSS is available in French and English. After you confirm your choice, the whole interface will be displayed in the chosen language.
+
+There are parts of FreshRSS that are not translated and are not intended to be translated. For now, the logs visible in the application as well as the one generated by the script of automatic update are part of it.
+
+## Theme
+
+In matters of taste and color, there can be no disputes. This is why FreshRSS offers six official themes:
+
+ * *Blue Lagoon* by **Mister aiR**
+ * *Dark* by **AD**
+ * *Flat design* by **Marien Fressinaud**
+ * *Origine* by **Marien Fressinaud**
+ * *Pafat* by **Plopoyop**
+ * *Screwdriver* by **Mister aiR**
+
+If none of these are suitable for you, it is always possible to create your own.
+
+To select a theme, simply scroll through the themes and select a theme that appears. After confirmation, the theme will be applied to the interface.
+
+## Content width
+
+There are some who prefer short lines of text while others prefer to maximize the available screen space. To satisfy the maximum number of people it is possible to choose the width of the displayed content. There are four settings available:
+
+ * **Fine** which displays content up to 550 pixels
+ * **Medium** which displays content up to 800 pixels
+ * **Large** which displays content up to 1000 pixels
+ * **No limit** which displays the content on 100% of the available space
+
+## Article icons
 
 **TODO**
 
-# Reading options
+## HTML5 notification timout
+
+After the automatic updates of the feeds, FreshRSS uses the HTML5 notification API to notify of the arrival of new articles.
+
+The duration of this notification can be set. By default, the value is 0.
+
+# Reading
 
 **TODO**
 
-# Archives
+# Archiving
 
 **TODO**
 
-# Shares
+# Sharing
 
 **TODO**
 
 # Shortcuts
 
 **TODO**
 
-# Filters
+# User queries
 
 **TODO**
 
 # Users
 
 **TODO**
 
-# Feed management
+## Authentication methods
+
+**Draft copy**
+
+### Authentication with Persona
+
+ 1. Persona, Mozilla authentication system: [https://login.persona.org/](https///login.persona.org/)
+ 2. Simply enter an e-mail address of your Persona
+ 3. It is possible to leave articles visible to visitors
+
+### HTTP Authentication
+
+ 1. Leaves nothing visible
```

Comment:
We should update this section to explain that the .htaccess must be placed in the `./i/` directory, so that the API, PubsubHubbub, etc. are working.